.packit.yml

```diff
@@ -0,0 +1,96 @@
+upstream_package_name: rhc
+downstream_package_name: rhc
+specfile_path: rhc.spec
+
+srpm_build_deps:
+  - gawk
+  - git-core
+  - golang
+  - make
+  - "pkgconfig(bash-completion)"
+  - "pkgconfig(dbus-1)"
+  - "pkgconfig(systemd)"
+  - "rpm_macro(forgemeta)"
+  - rpm-build
+
+actions:
+  post-upstream-clone:
+    - make dist
+    - make rhc.spec
+  get-current-version:
+    - awk '/^Version:/ {print $2;}' rhc.spec
+  create-archive:
+    - bash -c 'echo rhc-*.tar.*'
+  fix-spec-file:
+    - echo 'nothing to fix'
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    additional_repos:
+      - "copr://@yggdrasil/latest"
+    targets:
+      - centos-stream-9-aarch64
+      - centos-stream-9-x86_64
+      - rhel-8-aarch64
+      - rhel-8-x86_64
+      - rhel-9-aarch64
+      - rhel-9-x86_64
+
+  - job: copr_build
+    trigger: commit
+    additional_repos:
+      - "copr://@yggdrasil/latest"
+    branch: main
+    owner: "@yggdrasil"
+    project: latest
+    targets:
+      - centos-stream-9-aarch64
+      - centos-stream-9-x86_64
+      - rhel-8-aarch64
+      - rhel-8-x86_64
+      - rhel-9-aarch64
+      - rhel-9-x86_64
+
+  - job: tests
+    trigger: pull_request
+    identifier: "unit/centos-stream"
+    targets:
+      - centos-stream-9-aarch64
+      - centos-stream-9-x86_64
+    labels:
+      - unit
+    tf_extra_params:
+      environments:
+        - artifacts:
+            - type: repository-file
+              id: https://copr.fedorainfracloud.org/coprs/g/yggdrasil/latest/repo/centos-stream-$releasever/group_yggdrasil-latest-centos-stream-$releasever.repo
+
+  - job: tests
+    trigger: pull_request
+    identifier: "unit/rhel"
+    targets:
+      rhel-9-aarch64:
+        distros:
+          - RHEL-9-Nightly
+      rhel-9-x86_64:
+        distros:
+          - RHEL-9-Nightly
+      rhel-8-aarch64:
+        distros:
+          - RHEL-8-Nightly
+      rhel-8-x86_64:
+        distros:
+          - RHEL-8-Nightly
+    labels:
+      - unit
+    tf_extra_params:
+      environments:
+        - artifacts:
+            - type: repository-file
+              id: https://copr.fedorainfracloud.org/coprs/g/yggdrasil/latest/repo/rhel-$releasever/group_yggdrasil-latest-rhel-$releasever.repo
+          settings:
```

Comment:
```suggestion
        - settings:
```

Reply:
Is this a new schema from the packit file? We have this same dictionary setting in other packit.yaml files and they worked.